react/modules/tasks/forms.md

```diff
@@ -1,14 +1,19 @@
-# React forms.
+# React Forms
 
 ## Application Requirements
 
-1. Routing. There will be 3 routes:
-   - Main, should have links to other 2 routes
-   - Route for the form created using uncontrolled components approach
-   - Route for the similar form, but created using **React Hook Form**
-2. Redux. Use redux to store the data provided by both approaches on the Main route. You can use tiles to display data taken from each form.
-3. Forms
+1. **Portals**. Use React Portals to display forms in modals:
+
+   - Main page should have buttons to open 2 different modals
+   - Modal for the form created using uncontrolled components approach
+   - Modal for the similar form, but created using **React Hook Form**
+   - Modals should be properly accessible (focus management, ESC key to close, click outside to close)
+
+2. **Redux**. Use redux to store the data provided by both approaches on the Main route. You can use tiles to display data taken from each form.
```

Comment:
Shouldn't we allow using Zustand as an alternative for those, who picked it in the previous module?